### DISCUSSION TITLE
Flashing display for Ender3 S1

### DISCUSSION THREAD

**Discussion by KalleWetterberg** [mriscoc/Ender3V2S1#741] [Q&A] [answered]
Hi!

Had no issue flashing the Ender3 S1, love the new firmware. But I had issues displaying the mesh view with a bunch of glitches, lines everywhere. So I figured i'd flash the display aswell. But I can't do anything to make the screen flash the firmware.
I first tried with a 64GB microSD. Realized it had to be maximum 8GB. So I shrunk the size of it to 6GB, no luck either. So I ordered another microSD with 8GB size, still no luck.
And on every atempt i've followed the readme instructions. Format to FAT32 and 4k size. Put the folder in the root of the sd card and reconnected the display, and then power on the printer. But I get no reaction at all, just starts up as usual with the old display firmware.

I´ve tried both the "private" and "DWIN_SET" version, as i'm not sure which version of the printer I have. It should be the newer version "private", because it's a stock Ender 3 S1, that I got 2 weeks ago. But the insides is a bit different from the example pictures. I've attached a picture to showcase the inside. I've also tried the different styles, with both folders.

Couldn't find any other thread with the same issue, but i'm pretty new at Github, so sorry if I missed it.

 
![20230216_121237](https://user-images.githubusercontent.com/49005484/219351834-95afd456-140d-444f-9106-54e58b08ccb8.jpg)

**Comment by KalleWetterberg** [comment]
Forgot to mention, I tried to google the different numbers and texts on the board to find some help, without any luck. The text on the microchip is as follows: AIHMI T7 2142 N8V14203

**Comment by mriscoc** [comment, marked as answer]
Check your display here: https://github.com/mriscoc/Ender3V2S1/wiki/How-to-update-the-display
Seems to be a TJC, currently this screen has some [compatibility issues](https://github.com/mriscoc/Ender3V2S1/issues/542) with this firmware
